Dockerfile

```
@@ -8,7 +8,11 @@ COPY . ./
 RUN cd cmd/analyticproxy && go build; cd -; \
     cd cmd/medifor && go build; cd -
 
-FROM ubuntu:18.04
+FROM python:3.8-slim-buster
+COPY --from=builder /go/python /app/bin/
```

Not bad, but let's not use the builder image for this. The following will be easier to maintain:

```suggestion
RUN mkdir -p /src/medifor
ADD ./setup.py ./python /src/medifor/
```